_posts/2024-06-03-geoserver-tutorials.md

```diff
@@ -0,0 +1,94 @@
+---
+author: Milad Rafiei
+layout: post
+title: How to Implement Basic Security in Geoserver
+date: 2024-06-05
+categories:   
+- Tutorials
+---
+
+[GeoSpatial Techno](https://www.youtube.com/@geospatialtechno) is a startup focused on geospatial information that is providing e-learning courses to enhance the knowledge of geospatial information users, students, and other startups. The main approach of this startup is providing quality, valid specialized training in the field of geospatial information.
+
+( [YouTube](https://www.youtube.com/@geospatialtechno)
+| [LinkedIn](https://www.linkedin.com/in/geospatialtechno)
+| [Facebook](https://www.facebook.com/geospatialtechno)
+| [X](https://twitter.com/geospatialtechn)
+)
+
+----
+
+### How to Implement Basic Security in Geoserver
+In this session, we want to discuss the Security section in GeoServer, Defining Users, Groups, and Roles, and Granting rights to created users. If you want to access the complete tutorial, simply click on the [link](https://youtu.be/KCTGZJ2Trvw).
+
+[![](https://img.youtube.com/vi/KCTGZJ2Trvw/0.jpg)](https://www.youtube.com/watch?v=KCTGZJ2Trvw)
+
+## Introduction
+GeoServer has a robust security subsystem. Most of the security features are available through the Web Administration interface. In the Security panel, you can find links to set user properties and bind data to security rules. The basic idea is that you create users and roles, and then combine them with data to enable a specific set of access policies. You can also limit read and write access by role.
```

Comment:
...combine them with data **rules** to enable...

Reply:
It's done.